src/tests/pokete/test_director_3_0.py

```
@@ -0,0 +1,116 @@
+
+    def test_mutator_stacking(self):
+        # Force high anomaly chance
+        with patch.object(DifficultyManager, 'get_anomaly_chance', return_value=0.9):
+            # We iteration 100 times to ensure we see stacks
+            p = Poke("steini", 100, player=False)
+            # With 0.9 base chance and 0.4 multiplier, we expect ~2 mutators on average
+            # (0.9 for first, 0.36 for second, 0.14 for third)
+            self.assertGreaterEqual(len(p.mutators), 1)
+
+    @patch('pokete.classes.health_bar.HealthBar.update')
+    def test_vampiric_logic(self, mock_hb):
```

Ruff (0.14.14)

[warning] Unused method argument: `mock_hb`

(ARG002)